pylinalg/func/vector.py

```diff
@@ -476,4 +476,55 @@ def vector_make_spherical_safe(vector, /, *, out=None, dtype=None):
     return out
 
 
+def vector_euler_angles_from_quaternion(quaternion, /, *, out=None, dtype=None):
```

Comment:
Why is this prefixed with `vector` if it returns scalar angles? If this fits the name-rules that we have adopted, then its fine, but it looks a bit odd for me on first sight.

Reply:
Because it is a vector in the space of Euler space? I don't really have a good answer. I don't want to fuss about the names too much anymore as we'll rename everything to be less verbose soon anyway.

Reply:
👍